pkg/provider/managedk8s/ruleset/securityhardenedk8s/rules/2002_test.go

```diff
@@ -0,0 +1,103 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package rules_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	fakeclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	"github.com/gardener/diki/pkg/provider/managedk8s/ruleset/securityhardenedk8s/rules"
+	"github.com/gardener/diki/pkg/rule"
+)
+
+var _ = Describe("#2002", func() {
+	var (
+		client            client.Client
+		plainStorageClass *storagev1.StorageClass
+
+		ctx = context.TODO()
+	)
```

Comment:
```suggestion
	var (
		client            client.Client
		plainStorageClass *storagev1.StorageClass
		
		deleteReclaimPolicy    corev1.PersistentVolumeReclaimPolicy = "Delete"
		notDeleteReclaimPolicy corev1.PersistentVolumeReclaimPolicy = "NotDelete"

		ctx = context.TODO()
	)
```
I suggest we use these hard-coded policies in the units tests instead of `corev1.PersistentVolumeReclaimDelete` and `corev1.PersistentVolumeReclaimRetain`. So that we can catch when there are changes in upstream.